### PR TITLE
add 3.11 tests

### PR DESCRIPTION
**Pull request by dinhlongviolin1**

### CODE REVIEW

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   test-package:
     timeout-minutes: 20
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-20.04
```
Comment (FredLL-Avaiga) [inline .github/workflows/publish.yml]: Why going back to ubuntu 20 ? LTS is 22 at the moment

File: Pipfile [hunk 2]
```diff
@@ -43,7 +43,7 @@ pytest-cov = "*"
 pytest-mock = "3.6.1"
 tox = "==3.25.1"
 twine = "*"
-playwright = "<1.28"
```
Comment (FredLL-Avaiga) [inline Pipfile:46]: playwright over 1.28 is pulling a more recent version of gevent 
But I suppose that we're using the latest gevent now ?

File: setup.py [hunk 3]
```diff
@@ -53,7 +53,7 @@
         "python-magic>=0.4.24,<0.5;platform_system!='Windows'",
         "python-magic-bin>=0.4.14,<0.5;platform_system=='Windows'",
     ],
-    "arrow": ["pyarrow>=9.0,<10.0"],
+    "arrow": ["pyarrow>=10.0.1,<11.0"],
```
Comment (FredLL-Avaiga) [inline setup.py:56]: Should we update arrow to 10 on the front end side too then ?

**Review by FredLL-Avaiga — CHANGES_REQUESTED** [review]
ubuntu 22 is LTS

File: .github/workflows/publish.yml (hunk 1 above)
Comment (dinhlongviolin1) [reply .github/workflows/publish.yml]: yes I know. I am just testing to see why it is not passing the tests on github actions but my computer is passing

File: Pipfile (hunk 2 above)
Comment (dinhlongviolin1) [reply Pipfile:46]: we are using the latest now so I think the restriction can be lifted

File: setup.py (hunk 3 above)
Comment (dinhlongviolin1) [reply setup.py:56]: I will do it!

File: src/taipy/gui/gui.py [hunk 4]
```diff
@@ -734,10 +734,10 @@ def __send_ws(self, payload: dict) -> None:
                     payload,
                     to=self.__get_ws_receiver(),
                 )
-                time.sleep(0)
+                time.sleep(0.001)
```
Comment (FredLL-Avaiga) [inline src/taipy/gui/gui.py:737]: :-o

**Review by FredLL-Avaiga — APPROVED** [review]

File: src/taipy/gui/gui.py (hunk 4 above)
Comment (dinhlongviolin1) [reply src/taipy/gui/gui.py:737]: 0.001 is the default. If I dont put it then a warning will be shown in gevent 22 :D

**Review by FabienLelaquais — APPROVED** [review]
👌

**Pull request merged**